samples/scala-protobuf-eventsourced-customer-registry-subscriber/build-customer-service-image.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# this is script is used in CI to build scala-protobuf-eventsourced-customer-registry image
+# in CI we'll have a file kalix-sdk-version.txt at the home dir
+
+export SDK_VERSION=$(cat ~/kalix-sdk-version.txt)
+cd ../scala-protobuf-eventsourced-customer-registry 
+sbt -Dkalix-sdk.version=${SDK_VERSION} docker:publishLocal
+cd -
```

Comment:
That did the trick. 

I will add more comments in this file to explain why we need this indirection.